### DISCUSSION TITLE
How to make table background transparent

### DISCUSSION THREAD

**Discussion by brianmcdonald** [posit-dev/great-tables#151] [Q&A]
Thank you for making Great Tables. It's a fantastic tool.

I am trying to include a table in a quarto webpage that has a custom background color (beige). Is there a way to make the background of the table transparent or set the color? The default appears to be white. I could not find how to do it from the docs or examples.

**Comment by brianmcdonald** [comment]
I don't see an option to set a transparent background, but the background color can be set like this:
`gt_tbl.tab_options(table_background_color="lightcyan")` as highlighted in the recent blog post

**Reply by rich-iannone** [reply]
Could you use the `"transparent"` color here? That should be a valid CSS color.

**Reply by brianmcdonald** [reply]
“Transparent” doesn’t work unfortunately. it gives a “ValueError: “transparent” is not defined as a named color in css3”

**Reply by rich-iannone** [reply]
Thanks for verifying that. Will open an issue to ensure that `"transparent"` does work.
